pkg/vault/test/foundry/E2eSwap.t.sol

```diff
@@ -45,11 +45,17 @@ contract E2eSwapTest is BaseVaultTest {
     uint256 internal minSwapAmountTokenB;
     uint256 internal maxSwapAmountTokenB;
 
+    uint256 internal exactInOutDecimalsErrorMultiplier = 1;
+
     // We theoretically support the full range of token decimals, but tokens with extreme values don't tend to perform
     // well in AMMs, due to precision issues with their math. The lowest decimal value in common use would be 6,
     // used by many centralized stable coins (e.g., USDC). Some popular wrapped tokens have 8 (e.g., WBTC).
     uint256 private constant _LOW_DECIMAL_LIMIT = 6;
 
+    // Number of parameters specific to the pool type to be fuzzed. Raising this parameter will lower the amount of
+    // tested scenarios, so raise with caution.
+    uint256 internal constant NUMBER_PARAMETERS = 5;
```

Comment:
This got lowered from 10 to 5 while I was distracted :) I was going to comment on how 10 was really high, and would reduce the coverage. I wonder if there's a way to have two `fuzzPoolParam` functions, one with 2 (if that's the usual case), and one with 5, maybe introducing new short base contracts that define them, and you could use one or the other depending on what your pool needed. (Haven't thought this all the way through; just trying to keep our former coverage.)

Do we need to raise the number of tries then? Can we quantify how much this decreases the coverage?

Reply:
It doesn't decrease the coverage because the original fuzz tests (that don't fuzz the pool-specific parameters) remain. So, this will only affect the coverage of tests where the pool settings change (like fuzzing weights or alpha/beta).